src/main/jbake/content/contributing.md

```diff
@@ -21,25 +21,20 @@ a committer.
-No matter if you are a programmer or not, it is probably best to check JIRA first to figure out if the problem you identified is already
-known. If not, please create a JIRA issue in which you try to describe to the best of your knowledge the bug that you want to fix or the
-improvement that you would like to contribute. There are many recommendations on the Web on how to write a good bug report, like
-[Haje Jan Kamps' How to write the perfect bug report](https://medium.com/@Haje/how-to-write-the-perfect-bug-report-6430f5a45cd) blog post.
+The Apache Sling project organizes its "to do" list using the Apache [JIRA issue tracking system](https://issues.apache.org/jira/browse/SLING). No matter if you are a programmer or not, it is probably best to check JIRA first to figure out if the problem you identified is already known. If not, please create a JIRA issue in which you try to describe to the best of your knowledge the bug that you want to fix or the improvement that you would like to contribute. There are many recommendations on the Web on how to write a good bug report, like [Haje Jan Kamps' How to write the perfect bug report](https://medium.com/@Haje/how-to-write-the-perfect-bug-report-6430f5a45cd) blog post.
+
+If pull requests are familiar to you, the next step is to open one against [one of our modules](https://github.com/apache/?q=sling&type=all&language=&sort=). If creating a pull request from your own repository, setting the _[allow edits from maintainers](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)_ option is convenient as it allows us to update your fork as the pull request evolves.
 
-If pull requests are familiar to you, the next step is to open one against one of our modules. If creating a pull request from your
-own repository, setting the
-_[allow edits from maintainers](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork)_
-option is convenient as it allows us to update your fork as the pull request evolves.
+More details about how the project is structured in terms of repositories can be read on the [Getting and Building Sling](/documentation/development/getting-and-building-sling.html) page.
 
-More details about how the project is structured in terms of repositories can be read on the
-[Getting and Building Sling](/documentation/development/getting-and-building-sling.html) page.
+If you want to get involved into Sling development, we marked a number of issues in Jira with the label ["good-first-issue"](https://issues.apache.org/jira/browse/SLING-5767?jql=status%20%3D%20Open%20AND%20labels%20%3D%20easy-first-issue); we consider them a good starting point into Sling development. Just assign the ticket to you and create a pull request to address it. We provide you feedback and help you to get your contribution into Sling.
```

Comment:
The label says "good-first-issue" but the link says "easy-first-issue" :-)

Reply:
Also, you may want to search in the SLING project instead 

https://issues.apache.org/jira/issues/?jql=project%20%3D%20sling%20and%20status%20%3D%20open%20and%20labels%20%3D%20%22good-first-issue%22

Reply:
thank you, fixed